RevenueCatUI/CustomerCenter/Data/CustomerCenterConfigTestData.swift

```diff
@@ -144,7 +144,8 @@ enum CustomerCenterConfigTestData {
         expirationOrRenewal: .init(label: .nextBillingDate,
                                    date: .date("June 1st, 2024")),
         productIdentifier: "product_id",
-        store: .appStore
+        store: .appStore,
+        isLifetimeSubscription: false
```

Comment:
lifetime purchases are not subscriptions, so it would be more correct to use `isLifetimeProduct`